src/Core.UnitTests/SystemAbstractions/TimerFactoryTests.cs

```diff
@@ -22,7 +22,7 @@
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using SonarLint.VisualStudio.Core.SystemAbstractions;
 
-namespace SonarLint.VisualStudio.Integration.UnitTests
+namespace SonarLint.VisualStudio.Core.UnitTests
```

Comment:
Unrelated - moved existing TimeFactoryTests.cs from the integration test project to the core test project (should have been done previously when the product code was moved).